### DISCUSSION TITLE
[Bug]: Unable to automatically wrap after inserting pictures

### DISCUSSION THREAD

**Discussion by atomwh** [django-wiki/django-wiki#1351] [Bug]
### Describe the bug

The wiki content is like below
```
## test1
[image:1 align:left size:large]  

## test2
````
As expected, after inserting a picture, the following text can automatically wrap, but this is not the case. you can check the attachment image

![image](https://github.com/django-wiki/django-wiki/assets/25268385/87efb211-e9ac-44fd-b0d6-6717a1d99d7d)

### Steps to reproduce.

1. create a new wiki article
2. upload a image
3. write content like below
```
## test1
[image:1 align:left size:large]  

## test2
```

### Expected behavior

the text after image will automatically start in a new line

### Python Version

3.8

### Django Version

3.2

**Comment by benjaoming** [comment]
I'm not following the issue description - the image wraps next to the following text correctly.

What are you expecting?

**Reply by atomwh** [reply]
sorry to make you confused, what I am expecting is that the text after the inserted image will automatically start in a new line, like below

text before image
[image]
text after image

but now it's like this

text before image
[image] text after image

**Reply by oscarmcm** [reply]
@atomwh In markdown to make a jump between paragraphs those need to have a blank line between them:

for example:

```md
This is same paragraph text
and not jump is rendered
```

This is same paragraph text
and not jump is rendered

And this for example:

```md
this is one line

this is another line
```

this is one line

this is another line

**Reply by benjaoming** [reply]
Sorry, but I'm still not clear about what you are trying to do, @atomwh! Maybe you should just remove `align:left` from the code?

**Reply by atomwh** [reply]
thank you, remove `align:left` works

about the question, I think I should post more snapshot to explain.

* editor view
![image](https://github.com/django-wiki/django-wiki/assets/25268385/30684cbf-1cd5-44de-8349-46710ac6d612)

* preview view
![image](https://github.com/django-wiki/django-wiki/assets/25268385/4e441aff-e07e-4e56-9e6c-474429426fb4)

**Reply by atomwh** [reply]
And I also tried `<br>`
1. adding settings `WIKI_MARKDOWN_HTML_WHITELIST=['br']` and `WIKI_MARKDOWN_HTML_ATTRIBUTES=['br']
2. then add one `<br>` after  [image:1 align:left], here test2 sitll right after the image, not in the position under the image
3. after adding severn `<br>` ,  the text finally go to the position I wanted(under the image)

**Reply by atomwh** [reply]
I think I found the problem, while add align:left, it will add `float-left` style to figure, when I remove float-left, it behaved as expected.

```html
<div class="wiki-article">                                                         
  <h2 id="wiki-toc-test1">test1</h2>                                               
    <figure class="thumbnail float-left" style=""
      <a href="/media/wiki/images/2/cd3fc848f9d94eac823d8cab48951442/pexels-eberhard-grossgasteiger-1064162.jpg" class="cboxElement">
        <img src="/media/cache/8e/47/8e47dcf74f4cef18e828f439c73402b9.jpg" alt="pexels-eberhard-grossgasteiger-1064162.jpg">
      </a>
      <figcaption class="caption"></figcaption>
    </figure>
    <p></p>                                                                          
  <h2 id="wiki-toc-test2">test2</h2>                                                 
  </div>
```
Also when i add `<br style="clear:both"/>` before `<h2 id="wiki-toc-test2">test2</h2>`, test2 will start in a new line.

The behavior of `align:left` does not seem to comply with the Markdown specification. Hope you guys can fix it. Thank you.

**Reply by benjaoming** [reply]
In what way does `float-left` not comply with the Markdown spec of `align:left` ?

**Comment by atomwh** [comment]
When I use the following content, the layout now is as expected.
```
# test1
[image:1 align:left size:large]  

# test2
```
![image](https://github.com/django-wiki/django-wiki/assets/25268385/db81466a-0842-49a4-9aaf-067f922ed492)
